apps/web/src/components/Table/ApiManagedTable/useFilterOptions.test.tsx

```diff
@@ -116,7 +116,6 @@ describe("useFilterOptions", () => {
     it.skip("performs 3 API client queries", () => {
       // eslint-disable-next-line @typescript-eslint/no-unused-vars
       const result = renderHookWithProviders({});
-      // TODO: Refactor to access function mock and check if Urql client is being called appropriately.
       // expect(mockClient.executeQuery).toBeCalledTimes(3);
     });
```

Comment:
If we are not going to action the TODO would it be worth it to just drop this test entirely? Looks like it requires the TODO to be complete before we no longer skip it.